mycroft/enclosure/gui.py

```diff
@@ -228,9 +228,15 @@ def remove_pages(self, page_names):
         # Convert pages to full reference
         page_urls = []
         for name in page_names:
-            page = self.skill.find_resource(name, 'ui')
+            if name.startswith("SYSTEM"):
```

Comment:
The changes to this file seem unrelated, were they meant to be included in this PR? or is it a separate thing?

Reply:
Remove pages had an incomplete implementation and this fits fine with the PR, remove pages was not looking for the correct paths for SYSTEM_html.qml and any other qml file that is handled by resources in core, this fixes that issue